### ISSUE TITLE
Don't work with definitions in camtrap-dp-profile

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/camtrap-dp#122]
`camtrap-dp-profile` currently defines `organizations` and `species` in a `definitions`:

https://github.com/tdwg/camtrap-dp/blob/9d1d85a69f84435e2ad72ef0ef9f1de4f10d5f2a/camtrap-dp-profile.json#L6-L7

 And then references these definitions later. Other properties are defined in context. I would define all of them in context:

```json
"organizations": {
  "description": "List of organizations related to this data package.",
  "type": "array",
  "items": {
    "type": "object",
    "required": [
      "title",
      "url"
    ],
    "properties": {
      "title": {
        "description": "Organization name.",
        "type": "string",
        "format": "default"
```

**Comment by peterdesmet** [comment]
@kbubnicki are you fine with this?

**Comment by kbubnicki** [comment]
Good point! It will make `camtrap-dp-profile` easier to read.

**Comment by peterdesmet** [comment]
Closed in #130
